### ISSUE TITLE
Elasticsearch 6.2

### ISSUE THREAD

**Issue by arko-r** [ngageoint/elasticgeo#62]
From Elasticsearch 6.2, Elasticsearch has support for WKT. I tried to load some WKT points into Geoserver from Elasticsearch using ElasticGeo but the points are not visible on the map I created.

Could this be supported?

**Comment by sjudeng** [comment]
Yes it would be great to add support for this. We'll look into it in coming weeks or would welcome a pull request for the feature if you're interested in contributing.

**Comment by arko-r** [comment]
I will try to implement it in my spare time in the coming week.

**Comment by sjudeng** [comment]
Support for Elasticsearch 6.2 including WKT-formatted geoshapes will be added in #68. If you'd like to test off the branch in that PR (or in master after it's merged) please report any issues here.
